README.md

```diff
@@ -79,3 +83,9 @@ Jubatus Examples
 |13. winequality                      |           |             | :+1:       |      |       |         |       |        |Ru           |
 |14. slot                             |           |             |            |      |       |         |       |  :+1:  |Py/Ja        |   
 |15. iris                             | :+1:      |             |            |      |       |         |       |        |Py           |   
+|16. network\_intrusion\_detection    |           |             |            |      |       | :+1:    |       |        |Py/Ru        |
+
+Python を使用したサンプルについて
```

Comment:
This PR re-added this sentence (which was removed in #82); I fixed directly in 579e86e443f54f2c500120c4987974033b7bdf5a